src/controller/python/chip-device-ctrl.py

```diff
@@ -1096,7 +1171,6 @@ def start_rpc_server():
         server.register_function(zcl_command)
         server.register_function(zcl_add_network)
         server.register_function(zcl_enable_network)
-        server.register_function(resolve)
```

Comment:
What is the reason for this deletion? Seems like we still need and use the resolve RPC

Reply:
resolve command is duplicated, look at line:
```        server.register_function(resolve)```
https://github.com/chip-csg/connectedhomeip/blob/a8394cc0c027992372775cf81c5cf6f76a352e23/src/controller/python/chip-device-ctrl.py#L1168